core/tests/protractor_utils/ExplorationEditorPage.js

```diff
@@ -88,6 +88,8 @@ var ExplorationEditorPage = function() {
   var saveDiscardToggleButton = element(
     by.css('.protractor-test-save-discard-toggle'));
   var saveDraftButton = element(by.css('.protractor-test-save-draft-button'));
+  var navigateToTranslationTabButtonFeedback = element(
```

Comment:
I don't understand this var, can you please explain?

Reply:
This var stores the element of Feedback sub tab button in Translation Tab. Changing it to  TranslationTabFeedbackButton because its more relatable.
Thanks!